aws/table_aws_directory_service_directory.go

```diff
@@ -0,0 +1,301 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+
+	"github.com/aws/aws-sdk-go/service/directoryservice"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsDirectoryServiceDirectory(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_directory_service_directory",
+		Description: "AWS Directory Service Directory",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("directory_id"),
+			ShouldIgnoreError: isNotFoundError([]string{"InvalidParameterValueException", "ResourceNotFoundFault"}),
+			Hydrate:           getDirectoryServiceDirectory,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listDirectiryServiceDirectorties,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The fully qualified name of the directory.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "directory_id",
+				Description: "The directory identifier.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) that uniquely identifies the directory.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getDirectoryArn,
+				Transform:   transform.FromValue(),
+			},
+			{
+				Name:        "access_url",
+				Description: "The access URL for the directory, such as http://<alias>.awsapps.com.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "alias",
+				Description: "The alias for the directory.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "connect_settings",
```

Comment:
Please fix the ordering of columns, JSON column should come after string columns.